sourced/ml/__main__.py

```diff
@@ -72,12 +74,22 @@ def add_parser(name, help):
     repos2coocc_parser.set_defaults(handler=repos2coocc_entry)
     add_engine_args(repos2coocc_parser)
     add_repo2_args(repos2coocc_parser, quant=False)
+    add_split_stem_arg(repos2coocc_parser)
     repos2coocc_parser.add_argument(
         "-o", "--output", required=True,
         help="[OUT] Path to the Cooccurrences model.")
-    repos2coocc_parser.add_argument(
-        "--split-stem", default=False, action="store_true",
-        help="Split Tokens to parts (ThisIs_token -> ['this', 'is', 'token']).")
+    # ------------------------------------------------------------------------
+    repos2roles_and_ids = add_parser(
+        "repos2roles_ids", "Converts a UAST to a list of pairs, where pair is a role and "
+        "identifier. Role is merged generic roles where identifier was found.")
+    repos2roles_and_ids.set_defaults(handler=repos2roles_and_ids_entry)
+    add_engine_args(repos2roles_and_ids)
+    add_extractor_args(repos2roles_and_ids)
+    add_split_stem_arg(repos2roles_and_ids)
+    repos2roles_and_ids.add_argument(
+        "-o", "--output", required=True,
+        help="[OUT] Path to the directory where spark should store the result. "
+             "Inside the direcory you find result is csv format, status file and sumcheck files.")
```

Comment:
When we get out of this deadline clinch, we should add a plugin architecture and split `__main__` into automatically registered commands.